Added note about metric categories (#521)
Consensys/doc.teku: docs/reference/cli/index.md

  </TabItem>
</Tabs>

Categories for which to track metrics. Options are `JVM`, `PROCESS`, `BEACON`, `DISCOVERY`, `EVENTBUS`, `EXECUTOR`, `LIBP2P`, `NETWORK`, `STORAGE`, `STORAGE_HOT_DB`, `STORAGE_FINALIZED_DB`, `REMOTE_VALIDATOR`, `VALIDATOR`, `VALIDATOR_PERFORMANCE`, `VALIDATOR_DUTY`.  All but `VALIDATOR_DUTY` categories are enabled by default.

> When `metrics-categories` is used, only the categories specified in this option will be enabled (all other categories will be disabled).

Reviewer: ```suggestion
When `metrics-categories` is used, only the categories specified in this option are enabled (all other categories are disabled).
```
